modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -276,9 +278,15 @@ export class ClientAddressListMethods extends ClientCore
         return null;
       }
       // format in the metadata field
-      const metadataString = await this.ipfs.fetchString(addressListProposal.metadata);
-      // TODO: Parse and validate schema
-      const metadata = JSON.parse(metadataString) as ProposalMetadata;
+      let metadata: ProposalMetadata = UNSUPPORTED_PROTOCOL_PROPOSAL_METADATA;
+      try {
+        const metadataCid = resolveIpfsCid(addressListProposal.metadata);
+        const metadataString = await this.ipfs.fetchString(metadataCid);
+        // TODO: Parse and validate schema
+        metadata = JSON.parse(metadataString) as ProposalMetadata;
+      } catch (err) {
+        console.warn(err);
```

Comment:
Why printing things?

Reply:
Left it here for the end user to know that there is an error but its not fatal, I'll delete it if you want